hyppo/ksample/mean_embedding.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+from numba import jit
+from scipy.stats import chi2
+from ._utils import _CheckInputs
+from .base import KSampleTestOutput, KSampleTest
+
+
+class MeanEmbeddingTest(KSampleTest):
+    r"""
+        Mean Embedding test statistic and p-value
+
+        The Mean Embedding test is a two-sample test that uses
+        differences in (analytic) mean embeddings of two data
+        distributions in a reproducing kernel Hilbert space in order
+        to determine how different the data are.
+
+        Notes
+        ____
+
+        The test statistic, like the Smooth CF statistic, takes on the following form:
+
+        :math: 'nW_n\Sigma_n^{-1}W_n'
+
+        As seen in the above formulation, this test-statistic takes the same form as
+        the Hotelling :math: 'T^2' statistic. However, the components are
+        defined differently in this case. Given data sets
+        X and Y, define the following as :math: 'Z_i', the vector of differences:
+
+        :math: 'Z_i = (k(X_i, T_1) - k(Y_i, T_1), \ldots, k(X_i, T_J) - k(Y_i, T_J)) \in mathbb{R}^J'
+
+        The above is the vector of differences between kernels at test points, :math: 'T_j'. The kernel
+        maps into the reproducing kernel Hilbert space.
+        This same formulation is used in the Mean Embedding Test.
+        Moving forward, :math: 'W_n' can be defined:
+
+        :math: 'W_n = \frac{1}{n} \sum_{i = 1}^n Z_i
+
+        This leaves :math: '\Sigma_n', the covariance matrix as:
+
+        :math: '\Sigma_n = \frac{1}{n}ZZ^T'
+
+        Once :math: 'S_n' is calculated, a threshold :math: 'r_{\alpha}' corresponding to the
+        :math: '1 - \alpha' quantil of a Chi-squared distribution w/ J degrees of freedom
+        is chosen. Null is rejected if :math: 'S_n' is larger than this threshold.
+
```

Comment:
what about the class parameters documentation?

Reply:
added